src/care/RAJAPlugin.h

```diff
@@ -28,6 +28,7 @@ namespace chai {
 namespace care {
    class RAJAPlugin {
```

Comment:
We should be able to remove this class now that the functionality has been moved elsewhere.